easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-intel-2020b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'PyRETIS'
+version = '2.5.0'
+
+homepage = 'http://www.pyretis.org'
+description = """PyRETIS is a Python library for rare event molecular simulations with emphasis on methods based
+ on transition interface sampling and replica exchange transition interface sampling."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('matplotlib', '3.3.3'),
+    ('MDTraj', '1.9.5'),
+    ('PyTables', '3.6.1'),
+    ('tqdm', '4.56.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('colorama', '0.4.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/colorama/'],
+        'checksums': ['5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b'],
+    }),
+    (name, version, {
+        'source_tmpl': 'pyretis-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyretis'],
+        'checksums': ['b4b842a83f39e97a9bbf2d9797f53823b9e741b479b5a6744aeb42daf1c7ad2b'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/pyretisanalyse', 'bin/pyretisrun'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    "pyretisanalyse --help",
+    "pyretisrun --help",
+]
+
```

Comment:
```suggestion

sanity_pip_check = True

```

Reply:
I'm just gonna apply these since I don't think there is any need to discuss them.